src/main/java/uk/gov/dvsa/model/cvs/CvsTrlPRSBilingual.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.dvsa.model.cvs;
```

Comment:
I'm not sure the model is correct, I have just tried generating a TRL PRS cert in AWS and it is throwing an error as it does not recognise the PRSDefectsWelsh field.